_includes/corridors/usd-bank-egypt.md

```diff
@@ -0,0 +1,177 @@
+{% include corridors/recipient_name.md %}
+
+## USD::Bank
+
+For USD bank payments in Egypt please use:
+
+{% capture data-raw %}
+```javascript
+ "details": {
+  {{ recipient_name }},
+  "street": "1 Main Street",
+  "phone_number": "+201023456789",
+  "iban": "EG380019000500000000263180002",
+  "transfer_reason": "personal_account",
+}
```

Comment:
@cesarecamurani shouldn't we include country since its mandatory?
<img width="606" alt="Screenshot 2024-01-24 at 18 20 04" src="https://github.com/transferzero/api-documentation/assets/26022423/5d62aa1c-b955-46d0-a907-f7faf30f790a">